docs/architecture.md

```diff
@@ -0,0 +1,24 @@
+# Architecture
+
+## Domain Driven Design
+
+We follow the goal of domain driven design to keep a clean domain layer.<br>
+Business logic is handled in the domain layer.<br>
```

Comment:
I'd avoid the word "layer" Just say "domain"

Reply:
I want to clearly distinguis it from API and persistence. I think calling it a layer is correct.

Reply:
Hmm, I understood we were going for hexagonal hence layer is problematic.